docs/howtoguides/how_to_display_usns_that_affect_the_system.rst

```diff
@@ -0,0 +1,32 @@
+How to display USNs that affect the system
+******************************************
```

Comment:
Similarly to my comment above, can we shorten the file names of these files, e.g. "display-usns-affecting-system" or something like that

Reply:
Actually, on second thoughts - can we roll all three of these into one page of "how to use the pro vulnerability command"?

Reply:
Fair enough

Reply:
Discussed side-channel, leaving as is for now